Week2/Homework/task10.js

```diff
@@ -0,0 +1,27 @@
+// TASK-10
+
+// 10. Write a program to answer the following questions:
+// 10.1 Can you store multiple types in an array? Numbers
+// and strings? Make an example that illustrates your answer.
+// 10.2 Can you compare infinities? (Not in Eyad's world)
+// - does 6/0 === 10/0? How can you test this?
+// 10.3 Add console.log statements to the above program 
+// in which you show that you understand the concepts (just 
+// like you've done in the above assignments).
+
+let multipleTypedArray = [0, "string", "000", true];
```

Comment:
you can make this `let` a `const` because it's value won't change

Reply:
(And this applies to more parts of your code)

Reply:
Thank you for your comments, I've made some changes as you requested.